docs/book/src/topics/eks/addons.md

```diff
@@ -0,0 +1,59 @@
+# EKS Addons
+
+[EKS Addons](https://aws.amazon.com/blogs/containers/introducing-amazon-eks-add-ons/) can be used with EKS clusters created using Cluster API Provider AWS.
+
+Addons are supported in EKS clusters using Kubernetes v1.18 or greater. 
+
+## Installing addons
+
+To install an addon you need to declare them by specifying the name, version and optionally how conflicts should be resolved in the `AWSManagedControlPlane`. For example:
```

Comment:
How would conflicts arrise? Conflicts with what exactly?

Reply:
This is on the AWS API for addons and so I exposed it. Theres only a  single addon available at present so this may make more sense in the future.

Reply:
Yeah, just took a look at the AWS SDK
```
    // How to resolve parameter value conflicts when applying the new version of
    // the add-on to the cluster.
    ResolveConflicts *string `locationName:"resolveConflicts" type:"string" enum:"ResolveConflicts"`
```
Still not clear exactly how a conflict may arise but I'm not too worried about it at this stage.

Reply:
The AWS API uses server side apply in the implementation. This means for the CNI, for example, (the only addon atm) enabling the addon can fail if say the `image` field is managed by some other `manager` (the addons API manages as `eks`) but will ignore this if you set `overwrite`.